src/components/tree-view.tsx

```diff
@@ -139,13 +174,15 @@ export const TreeView = ( { tree, setTree }: TreeViewProps ) => {
 	};
 
 	return (
-		<div>
+		<div role="tree">
 			{ tree.map( ( node, index ) => (
 				<TreeItem
 					key={ node.id }
 					node={ node }
 					onPatchNode={ handlePatchNode }
-					level={ 0 }
+					level={ 1 }
+					index={ index }
+					setsize={ tree.length }
```

Comment:
I see that `setsize` is the name for aria, but we should not depend on names where it's used. In our codebase it would be clear to show that `setsize` is the number of all siblings, otherwise it's not clear what `set` means.
```suggestion
					siblingsLength={ tree.length }
```